### PR TITLE
Index Page Context Updates

### PR DESCRIPTION
**Pull request by zstix**
## Description
Updates to the index page copy / button.

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -108,9 +109,9 @@ const IndexPage = ({ data, pageContext }) => {
               <Button
                 as={ExternalLink}
                 variant={Button.VARIANT.PRIMARY}
-                href="https://newrelic.com/signup?partner=Developer+Edition"
+                href="https://newrelic.com/signup/?utm_source=developer-site"
```
Comment (jerelmiller) [inline src/pages/index.js:112]: It looks like the `/signup/` page (with the trailing `/`) doesn't work. Can you remove the trailing `/`?

Comment (zstix) [reply src/pages/index.js:112]: Ah, copied and pasted from the doc. Kinda surprised that doesn't work. Either way, updating it now (via suggestion)
```suggestion
                href="https://newrelic.com/signup?utm_source=developer-site"
```

Comment (jerelmiller) [reply src/pages/index.js]: Ya not sure why it doesn't work. I happened to see something mentioned in one of the slack channels.

**Comment by jerelmiller** [comment]
The PR preview built. Not sure why its stuck in pending state here. Going to go ahead and merge :)

**Pull request merged**